### DISCUSSION TITLE
How are you deploying monorepos?

### DISCUSSION THREAD

**Discussion by thebetterjort** [vercel/turborepo#385] [Help] [answered]
Reading [Netlify Documentation](https://docs.netlify.com/configure-builds/common-configurations/monorepos/#build-multiple-sites-from-a-monorepo) and it looks like we could create monorepos with turborepo and deploy using Netlify. Does anybody have any guidance on how they are deploying monorepos? Any examples on Vercel would be awesome too.

**Comment by baptisteArno** [comment]
Check [this](https://vercel.com/docs/concepts/git/monorepos#turborepo), it should be the same on Netlify

**Comment by anthonyshew** [comment, marked as answer]
Various CI integrations can be found [here](https://turbo.build/repo/docs/ci).
